File: decompiled/sources/com/hjq/http/request/TraceRequest.java
package com.hjq.http.request;

import androidx.lifecycle.LifecycleOwner;
import com.hjq.http.model.HttpMethod;

public final class TraceRequest extends UrlRequest<TraceRequest> {
    public TraceRequest(LifecycleOwner lifecycleOwner) {
        super(lifecycleOwner);
    }

    /* access modifiers changed from: protected */
    public String getRequestMethod() {
        return HttpMethod.TRACE.toString();
    }
}

File: decompiled/sources/com/hjq/http/request/TraceRequest.kt
package com.hjq.http.request

import androidx.lifecycle.LifecycleOwner
import com.hjq.http.model.HttpMethod

class TraceRequest(
    lifecycleOwner: LifecycleOwner
) : UrlRequest<TraceRequest>(lifecycleOwner) {
    override fun getRequestMethod(): String {
        return HttpMethod.TRACE.toString()
    }
}
